recipes/tensorstore/build.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [[ $target_platform =~ osx.* ]]; then
+    CLANG="$CC_FOR_BUILD" source gen-bazel-toolchain
+    # SDK's curl gets in the way
+    if [[ -d "$CONDA_BUILD_SYSROOT/usr/include/curl" ]]; then
+        mv "$CONDA_BUILD_SYSROOT/usr/include/curl" "$CONDA_BUILD_SYSROOT/usr/include/curl.do-not-use"
+    fi
+else
+    source gen-bazel-toolchain
+fi
+
+system_libs="com_google_boringssl"
+system_libs+=",org_sourceware_bzip2"
+system_libs+=",org_blosc_cblosc"
+system_libs+=",se_curl"
+system_libs+=",jpeg"
+system_libs+=",png"
+system_libs+=",org_lz4"
+system_libs+=",org_tukaani_xz"
+system_libs+=",net_zlib"
+system_libs+=",com_github_pybind_pybind11"
+system_libs+=",com_github_nlohmann_json"
+system_libs+=",org_aomedia_avif"
+# system_libs+=",com_google_absl"
+export TENSORSTORE_SYSTEM_LIBS="$system_libs"
+
+build_options=""
+build_options+=" --crosstool_top=//bazel_toolchain:toolchain"
+build_options+=" --logging=6"
+build_options+=" --verbose_failures"
+build_options+=" --toolchain_resolution_debug"
+build_options+=" --local_cpu_resources=${CPU_COUNT}"
+build_options+=" --subcommands"  # comment out for debugging
+export TENSORSTORE_BAZEL_BUILD_OPTIONS="$build_options"
+
+# TODO: figure out why we need both TENSORSTORE_BAZEL_BUILD_OPTIONS and a bazelrc
```

Comment:
If we supply the arguments twice, we should make sure to supply the identical ones. At least make them identical here. I would guess though that once should be enough.

Reply:
@isuruf and I talked about the difference and apparently this configuration is the only one that works. Making them identical fails 🤷

Reply:
Yes, and I tried not setting `TENSORSTORE_BAZEL_BUILD_OPTIONS` too and it didn't work as well.